### DISCUSSION TITLE
No way to access $HOME in the vitest.nodeExecutable setting

### DISCUSSION THREAD

**Discussion by ethanjdiamond** [vitest-dev/vscode#436] [Feedback]
I'm trying to set up the settings.json for my project so `vitest.nodeExecutable` points at the local machine's mise shim for node, but I can't seem to find a way to do it.

I tried this: `vitest.nodeExecutable="~/.local/share/mise/shims/node"`
But it resolves to `[PATH_TO_WORKSPACE]~/.local/share/mise/shims/node` instead for `[HOME]/.local/share/mise/shims/node`.

Without this functionality, I won't be able to set this up for all users of my project properly. Is there a way to do this?

**Comment by sheremet-va** [comment]
You should be able to use `~/` in the next version (1.0.4)
